src/esp/sim/Simulator.cpp

```diff
@@ -1076,6 +1076,59 @@ scene::SceneNode* Simulator::loadAndCreateRenderAssetInstance(
       assetInfo, creation, sceneManager_.get(), tempIDs);
 }
 
+#ifdef ESP_BUILD_WITH_VHACD
+std::string Simulator::convexHullDecomposition(
+    const std::string& filename,
+    const assets::ResourceManager::VHACDParameters& params,
+    const bool renderCHD,
+    const bool saveCHDToObj) {
+  Cr::Utility::Debug() << "VHACD PARAMS RESOLUTION: " << params.m_resolution;
+
+  // generate a unique filename
+  std::string CHDFilename =
```

Comment:
```suggestion
  std::string chdFilename =
```

naming policy: variables should start with lowercase; same elsewhere